go.mod

```diff
@@ -4,8 +4,8 @@ go 1.15
 
 require (
 	github.com/buraksezer/olric v0.3.11
-	github.com/dgraph-io/badger/v3 v3.2103.1
-	github.com/dgraph-io/ristretto v0.1.0
+	github.com/dgraph-io/badger/v3 v3.2011.1
+	github.com/dgraph-io/ristretto v0.0.4-0.20210122082011-bb5d392ed82d
```

Comment:
Downgrading Badger v3 prevents Caddy from building? That isn't due to the prior commits [build failure](https://github.com/Darkweak/Souin/runs/3358600607) was it? That was about an RC version of Badger v2 AFAIK.

At least when I used xcaddy I used `--with` to override for latest v2 release and it build successfully avoiding the same build failure error I linked which was experienced in my bug report.

There was some nice improvements in the [newer releases of Badger v3](https://github.com/dgraph-io/badger/releases), if there was another issue that you experienced with building Caddy can you reference it and any related Github issue to track?